src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,13 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    static final Color[] colorValues = Color.values();
```

Comment:
The constant `colorValues` should have an access modifier to specify its visibility explicitly and its name should be in uppercase to follow Java naming conventions for constants.